build.sh

```diff
@@ -19,10 +19,9 @@ function extract_microcode()
                 echo "Failed to extract microcode"
                 rm -rf 7D25v13*
         else
-                echo "UEFIExtract files found."
-                echo "Install it from https://github.com/LongSoft/UEFITool/tree/new_engine"
-                echo "and put into a directory exporeted by PATH variable to"
-                echo "be able to extract microcode"
+                echo "UEFIExtract not found, but it's required to extract microcode!"
+                echo "Install it from: https://github.com/LongSoft/UEFITool/releases/download/A59/UEFIExtract_NE_A59_linux_x86_64.zip"
+                exit 1
```

Comment:
I didn't want to hard exit here as somebody may want to put microcode without UEFIExtract. The IF block later will detect the lack of microcode.

Reply:
@miczyg1 Understood, can you change that and merge if no other comments?